tests/light/src/driver_io/network/network_io.py

```diff
@@ -45,8 +44,6 @@ def __init__(self, ip, port, transport, ip_proto_version=None):
         self.__server = None
         self.__message_reader = None
 
-        atexit.register(self.stop_listener)
```

Comment:
If at all possible I'd squash this commit with the following one, as you are removing one mechanism in place for another and I'd argue that this replacement was more apparent if it was in the same patch.

Reply:
Thank you very much for the review comment. 
Agreed and I will fix it.

Reply:
In my last push I have fixed this